Captain-AJ.md

```diff
@@ -0,0 +1,54 @@
+# name: "@Captain-AJ"
+# project: "An autonomous butler robot"
+
+# Project Name
+A.L.F.R.E.D. Butlerbot
+
+## Summary
+
+I plan to build an Arduino controlled autonomous robot butler named A.L.F.R.E.D. (Advanced, Little, Fit, Robot, Education, Device) that can be summoned at the press of a 
+button and deliver items. I am excited because this practical project has been on my list for sometime now. 
+After a long part search, my efforts to build this contraption summed in the listed Devastator robot kit. The set includes treads, metal motors, an aluminum tank chassis, 
+and a battery compartment.
+All coincides with my desire to apply computer and mechanical engineering concepts to develop a computing module in physical excellence.
+
+## Plan
+
+1. Order kit and required parts.
+
+2. Read a sizable amount of Arduino Cookbook to become well endowed with appropriate knowledge.
+
+3. Build the kit according to the instructions: 
+a. Assemble the tracks with gear components. 
+b. Assemble the robot's chassis and attach the Romeo BLE.
+c. Attach all sensors to the body, e.g., Ultrasonic sensor, speaker, etc.
+d. Pan/tilt systems for ultrasonic sensor and camera
+e. Wire all components 
+f. Attach the food storage unit.
+
+4. Programing (Python, C language):
+a. PixyCamera for color and facial recognition.
+b. Ultrasonic sensor for object detection and avoidance.
```

Comment:
would this be autonomous of for when you're controlling the robot? if you're controlling it, how would you control it? if it's autonomous, how do you plan on making it autonomous?

Reply:
One will load food into the storage unit upon the robot’s arrival at various intake points. I plan to incorporate a fully automatic procedure in the future. When enabled, the robot will be summoned via Bluetooth.

**Here is a list of functions:**

Button one: Instructs the robot to travel to a loading point.

Button two: Activates autonomous reference tracking for station one.

Button three: Activates autonomous reference point tracking for station two.

Button four: Activates autonomous reference tracking for station three.

Button five: Instructs the robot to return to resting point.

Hand gestures: These can be used as an alternative to the buttons.

Control pad/arrow keys: For manual control.

When the robot recognizes one’s face, it will say hello and the user's name.

After a certain amount of time, the robot will ask if the user is satisfied. Upon asking thrice, the bot will autonomously return to it's resting space, but, if the “satisfied” button is pressed, it will return earlier. 

When waiting to be loaded the bot will say, “waiting for load”, then, upon pressing buttons two-five, it will go to a specified drop-off point.

Programming:

Libraries: 

Pixy2: [https://github.com/charmedlabs/pixy2/tree/master/releases/arduino](https://github.com/charmedlabs/pixy2/tree/master/releases/arduino)

For vision processing: OpenCV

I plan to connect the sensors through the following means:

Gyro: I2C

Ultrasonic: PWM

Pixy2 Camera: SPI

Speaker: Digital

Display: SPI

Reply:
this is perfect, thank you sm! could you please add this to your file?

Reply:
I have added this to the file.